### PR TITLE
fix(griffin): fixed invalid column error when where clause in join SQL references single table

### PR DESCRIPTION
**Pull request by bluestreak01**

### CODE REVIEW

File: core/src/test/java/io/questdb/cutlass/pgwire/PGJobContextTest.java [hunk 1]
```diff
@@ -2650,6 +2650,7 @@ public void testRollbackDataOnStaleTransaction() throws Exception {
     }
 
     @Test
+    @Ignore
```
Comment (ideoma) [inline core/src/test/java/io/questdb/cutlass/pgwire/PGJobContextTest.java:2653]: Why ignore?

Comment (bluestreak01) [reply core/src/test/java/io/questdb/cutlass/pgwire/PGJobContextTest.java:2653]: I knew you'd ask. This test does not do what it advertises. It is corrupt. It is meant to test bind variable interaction but insteat it sends wrong username/password and never gets to test anything. Instead it hangs occasionally.

**Comment by ideoma** [comment]
I tried this on the branch


create table l as
( select x from long_sequence(100) )


create table r as
( select x + 50 from long_sequence(100) )

select x 
from l join r on l.x = r.x

and got back

Invalid column: r.x

**Comment by bluestreak01** [comment]
this is another bug altogether

**Review by ideoma — APPROVED** [review]

**Pull request merged**